### PR TITLE
Release AD777x

### PR DESCRIPTION
**Pull request by jansunil**
Release AD7134 project with the STM32H563 support

### CODE REVIEW

File: projects/ad777x_iio/scripts/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 numpy==1.22.4
-pyadi-iio==0.0.15
+pyadi-iio==0.0.12
```
Comment (D-Disha) [inline projects/ad777x_iio/scripts/requirements.txt:2]: Can this update be mentioned in the commit msg?

Comment (jansunil) [reply projects/ad777x_iio/scripts/requirements.txt:2]: Yes, I have updated the message now

**Comment by ribdp** [comment]
Should we update the copyright years in these files as well?

**Comment by jansunil** [comment]
Changelog: 
Updated copyright years
Updated extsettings to include the dma files

**Comment by jansunil** [comment]
Hi All- do these changes look good for a merge?

**Review by SaikiranGudla — APPROVED** [review]

**Review by ribdp — APPROVED** [review]

**Pull request merged**